images/ad-server/install-sambacc.sh

```diff
@@ -3,18 +3,44 @@
 set -ex
 
 wheeldir="$1"
-container_json_file="$2"
 
 wheeldir=/tmp
-wheel="$(find "${wheeldir}" -type f -name 'sambacc-*.whl')" \
+find "${wheeldir}"
+wheel="$(find "${wheeldir}" -type f -name 'sambacc-*.whl')"
+rpmfile="$(find "${wheeldir}" -type f -name '*sambacc-*.noarch.rpm')"
 
-if ! [ "$(echo "$wheel" | wc -l)" = 1 ]; then
-    echo "more than one wheel file found"
+rpmcount="$(echo "$rpmfile" | grep -c rpm)"
```

Comment:
When I tried to build with those changes on my machine (Fedora35), it fails on this line. The exit code of `grep` when there is no rpm is 1, which in turn causes this shell script to fail due to the `set -ex` at line 3.

Reply:
You are correct: as soon as I deleted my older sambacc image, everything works fine. At least it helped catch this one!
Other than that, LGTM.